### PR TITLE
Update settingup-project.mdx

### PR DESCRIPTION
**Pull request by Mariana-Caetano**
## What's the purpose of this pull request?

This PR reviews the [Setting Up the Project](https://evergreen.faststore.dev/docs/getting-started/settingup-project) doc.

## How it works?

This general review of the published documentation focuses on clarity and fixing typos.
🔗 [Preview](https://faststore-site-git-education-review-faststore.vercel.app/docs/getting-started/settingup-project)

### CODE REVIEW

File: apps/site/pages/docs/getting-started/settingup-project.mdx [hunk 1]
```diff
@@ -90,22 +90,27 @@ Before cloning the store's repository, ensure that you have the following tools
 </Callout>
 
 ## Step 1: Cloning the Store's Repository
-1. Clone your store's repository that was created during the [Onboarding](/docs/getting-started/onboarding#repository-and-live-store-created) by running the following command in the terminal:
+1. Open your terminal and navigate to the directory where you want to store your project.
+2. Clone your store's repository that was created during the [Onboarding](/docs/getting-started/onboarding#repository-and-live-store-created) by running the following command in the terminal:
 ```bash
     git clone https://github.com/vtex-sites/[store-name]
```
Comment (carolinamenezes) [inline apps/site/pages/docs/getting-started/settingup-project.mdx:96]:
```suggestion
    git clone https://github.com/vtex-sites/{storeName}
```

File: apps/site/pages/docs/getting-started/settingup-project.mdx [hunk 2]
```diff
@@ -90,22 +90,27 @@ Before cloning the store's repository, ensure that you have the following tools
...
 ```
 <Callout type="warning" emoji="⚠️">
 Replace `[store-name]` for your store's repository.
```
Comment (carolinamenezes) [inline apps/site/pages/docs/getting-started/settingup-project.mdx:99]:
```suggestion
Replace `{storeName}` for your store's repository.
```

File: apps/site/pages/docs/getting-started/settingup-project.mdx [hunk 3]
```diff
@@ -90,22 +90,27 @@ Before cloning the store's repository, ensure that you have the following tools
...
 </Callout>
 
-2. Change into the working directory (e.g., cd `faststore.store`) and install all the dependencies listed within the `package.json` file:
+3. Change into the working directory (e.g., cd `faststore.store`) by running the following command: 
 
 ```bash
 cd faststore.store
```
Comment (carolinamenezes) [inline apps/site/pages/docs/getting-started/settingup-project.mdx:105]:
```suggestion
cd {storeName}
```

File: apps/site/pages/docs/getting-started/settingup-project.mdx [hunk 4]
```diff
@@ -90,22 +90,27 @@ Before cloning the store's repository, ensure that you have the following tools
 </Callout>
 
 ## Step 1: Cloning the Store's Repository
-1. Clone your store's repository that was created during the [Onboarding](/docs/getting-started/onboarding#repository-and-live-store-created) by running the following command in the terminal:
+1. Open your terminal and navigate to the directory where you want to store your project.
```
Comment (carolinamenezes) [inline apps/site/pages/docs/getting-started/settingup-project.mdx:93]:
```suggestion
1. Open the terminal and navigate to the directory where you want to store your project.
```

File: apps/site/pages/docs/getting-started/settingup-project.mdx [hunk 5]
```diff
@@ -90,22 +90,27 @@ Before cloning the store's repository, ensure that you have the following tools
...
+2. Clone your store's repository that was created during the [Onboarding](/docs/getting-started/onboarding#repository-and-live-store-created) by running the following command in the terminal:
```
Comment (carolinamenezes) [inline apps/site/pages/docs/getting-started/settingup-project.mdx:94]:
```suggestion
2. Clone the repository created during the [Onboarding](/docs/getting-started/onboarding#repository-and-live-store-created) by running the following command in the terminal:
```

**Review by hellofanny — APPROVED** [review]
Nice ⭐ 


@Mariana-Caetano since this is a new section, should we start counting from 1 here?
image

**Comment by Mariana-Caetano** [comment]
> Nice ⭐
> 
> @Mariana-Caetano since this is a new section, should we start counting from 1 here? image

For sure, @hellofanny. Thanks! 💯

**Pull request merged**